src/components/SocialAuthPage.jsx

```diff
@@ -11,4 +22,9 @@ class SocialAuthPage extends Component {
   }
 }
 
-export default SocialAuthPage;
+SocialAuthPage.propTypes = {
+  socialMediaLogin: PropTypes.func.isRequired,
+  history: PropTypes.oneOfType([PropTypes.object]).isRequired
```

Comment:
There's only one option in the prop types here so why not simply make it:
```suggestion
  history: PropTypes.object.isRequired
```